config_models/models.py

```diff
@@ -102,7 +102,7 @@ class Meta(object):
 
     change_date = models.DateTimeField(auto_now_add=True, verbose_name=_("Change date"))
     changed_by = models.ForeignKey(
-        User,
+        settings.AUTH_USER_MODEL,
```

Comment:
Did you happen to run a makemigrations on this to see if this will generate new migrations in downstream apps? Seemingly inconsequential changes to models sometimes do that.

Reply:
@bmedx just tested - it doesn't change anything for existing apps that use auth.User.